bindings/go/osvschema/vulnerability.go

```diff
@@ -0,0 +1,170 @@
+package osvschema
+
+import (
+	"encoding/json"
+	"time"
+)
+
+// Package identifies the affected code library or command provided by the
+// package.
+//
+// See: https://ossf.github.io/osv-schema/#affectedpackage-field
+type Package struct {
+	Ecosystem string `json:"ecosystem"      yaml:"ecosystem"`
+	Name      string `json:"name"           yaml:"name"`
+	Purl      string `json:"purl,omitempty" yaml:"purl,omitempty"`
+}
+
+// Event describes a single version that either:
+//
+//   - Introduces a vulnerability: {"introduced": string}
+//   - Fixes a vulnerability: {"fixed": string}
+//   - Describes the last known affected version: {"last_affected": string}
+//   - Sets an upper limit on the range being described: {"limit": string}
+//
+// Event instances form part of a “timeline” of status changes for the affected
+// package described by the Affected struct.
+//
+// See: https://ossf.github.io/osv-schema/#affectedrangesevents-fields
+type Event struct {
+	Introduced   string `json:"introduced,omitempty"    yaml:"introduced,omitempty"`
+	Fixed        string `json:"fixed,omitempty"         yaml:"fixed,omitempty"`
+	LastAffected string `json:"last_affected,omitempty" yaml:"last_affected,omitempty"`
+	Limit        string `json:"limit,omitempty"         yaml:"limit,omitempty"`
+}
+
+// Range describes the affected range of given version for a specific package.
+//
+// See: https://ossf.github.io/osv-schema/#affectedranges-field
+type Range struct {
+	Type             RangeType              `json:"type"                        yaml:"type"`
+	Events           []Event                `json:"events"                      yaml:"events"`
+	Repo             string                 `json:"repo,omitempty"              yaml:"repo,omitempty"`
+	DatabaseSpecific map[string]interface{} `json:"database_specific,omitempty" yaml:"database_specific,omitempty"`
+}
+
+// Severity is used to describe the severity of a vulnerability for an affected
```

Comment:
Standardize for consistency.

```suggestion
// Severity describes the severity of a vulnerability for an affected
```